best-practices/README.md

```diff
@@ -107,7 +107,8 @@ Testing
 -------
 
 * Avoid `any_instance` in rspec-mocks and mocha. Prefer [dependency injection].
-* Avoid `its`, `let`, `let!`, `specify`, `before`, and `subject` in RSpec.
+* Avoid `its`, `let`, `let!`, `specify`, and `before` in RSpec.
+* Avoid using `subject` explicitly in RSpec `it` blocks.
```

Comment:
I'm not sure that the problem is people using `subject` in `it` blocks -- is that even possible?

How about something like this?
```
Avoid using `subject` in RSpec, except for groups of one-line specs.
```